modules/swagger-codegen/src/test/java/io/swagger/codegen/DefaultGeneratorTest.java

```diff
@@ -221,6 +222,32 @@ public void testGenerateUniqueOperationIds() {
         }
     }
 
+    @Test
+    public void testGenerateRubyClientWithHtmlEntity() throws Exception {
+        final File output = folder.getRoot();
+
+        final Swagger swagger = new SwaggerParser().read("src/test/resources/2_0/pathWithHtmlEntity.yaml");
+        CodegenConfig codegenConfig = new RubyClientCodegen();
+        codegenConfig.setOutputDir(output.getAbsolutePath());
+
+        ClientOptInput clientOptInput = new ClientOptInput().opts(new ClientOpts()).swagger(swagger).config(codegenConfig);
+
+        DefaultGenerator generator = new DefaultGenerator();
+        generator.opts(clientOptInput);
+        List<File> files = generator.generate();
+        boolean apiFileGenerated = false;
+        for (File file : files) {
+          if (file.getName().equals("default_api.rb")) {
+            apiFileGenerated = true;
+            // Ruby client should set the path unescaped in the api file
+            assertTrue(FileUtils.readFileToString(file, StandardCharsets.UTF_8).contains("local_var_path = \"/foo=bar\""));
+          }
+        }
+        if (!apiFileGenerated) {
+          fail("Default api file is not generated!");
+        }
+    }
+
```

Comment:
@cliffano thanks for writing the test case to cover the change. If it's not too much work, I would suggest we put the test under a new Java class in the [ruby folder](https://github.com/cliffano/swagger-codegen/tree/8bacbfb6916cf2a3cb8b9e69d62d9e49fda37fa2/modules/swagger-codegen/src/test/java/io/swagger/codegen/ruby). 

Ref: [Java test cases](https://github.com/cliffano/swagger-codegen/tree/8bacbfb6916cf2a3cb8b9e69d62d9e49fda37fa2/modules/swagger-codegen/src/test/java/io/swagger/codegen/java)